src/cookbooks/data_quality/issues_management.md

```diff
@@ -0,0 +1,50 @@
+## Issues management
+
+The `Issues` tab filtering issues by parameters like severity or date, and reviewing details such as impacted metrics and tables. 
+Users can also resolve, mute, or dismiss issues, helping maintain healthy data pipelines.
+
+
+<img src="../../assets/Bigeye/Issues-tab.png" style="max-width: 100%; height: 500px; width: 100%">
```

Comment:
You shouldn't need the manual styling and can use markdown instead.
Please also include alt text:

```suggestion
![The Issues overview on BigEye, allowing to filter issues](../../assets/Bigeye/Issues-tab.png)
```

Same applies to the images below.